marshmallow/validate.py

```diff
@@ -214,10 +216,27 @@ class Length(Range):
     message_min = 'Shorter than minimum length {min}.'
     message_max = 'Longer than maximum length {max}.'
     message_all = 'Length must be between {min} and {max}.'
+    message_equal = 'Length must be {equal}.'
+
+    def __init__(self, min=None, max=None, error=None, equal=None):
+        super(Length, self).__init__(min, max, error)
+        self.equal = equal
```

Comment:
It might make sense to raise a `ValueError` if equal is passed and either `min` and `max` is not `None` .